.ci/dockerfiles/Dockerfile.xpu

```
@@ -0,0 +1,134 @@
+RUN apt-get update -y && \
+    apt-get install -y build-essential cmake pkg-config autoconf libtool libcjson-dev && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# B70 XPUs require a newer version of following packages than what may be available in the base image.
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends --fix-missing libze1 libze-dev libze-intel-gpu1 intel-opencl-icd libze-intel-gpu-raytracing \
+    intel-ocloc intel-oneapi-compiler-dpcpp-cpp-2025.3 && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install xpu-smi
+RUN wget https://github.com/intel/xpumanager/releases/download/v1.3.6/xpu-smi_1.3.6_20260206.143628.1004f6cb.u24.04_amd64.deb -O /tmp/xpu-smi.deb && \
```

Hadolint (2.14.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)